### ISSUE TITLE
Panic when one forgets to `use pervasive::option`

### ISSUE THREAD

**Issue by jaylorch** [verus-lang/verus#383]
The verifier panics when one uses `Option` but forgets to `use pervasive::option`. For instance, when verifying the following code:

```
#[allow(unused_imports)]
use builtin_macros::*;
#[allow(unused_imports)]
use builtin::*;

#[allow(unused_attributes)]
mod pervasive;
//#[allow(unused_imports)]
//use pervasive::option::{Option, Option::None, Option::Some}; //<-- commenting this out crashes the verifier

verus! {

fn noneify() -> Option<i32>
{
    None
}

fn main() {
}

} // verus!
```

it panics with the following output:
```
thread 'rustc' panicked at 'constructor of undefined datatype', vir/src/well_formed.rs:144:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by tjhance** [comment]
This is definitely a bug. FYI, though, you can now use the `pervasive::prelude::*` to import Option, all the builtins, and a few other common types.

**Comment by tjhance** [comment]
fixed in [df5fa8a](https://github.com/verus-lang/verus/commit/df5fa8a733c306db24f2f00a2089e791d0c16cb8)
